GestioneOrari/src/Elaborato/Elaborato.java

```diff
@@ -1,8 +1,23 @@
 package Elaborato;
 
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import row.Row;
+
 public class Elaborato {
+	private List<Row> rowsList;
```

Comment:
List non è necessario di solito quando si mette un plurale a un campo si tratta di una collection in generale, quindi ti direi di togliere List dal nome.

```suggestion
	private List<Row> rows;
```